Bottom: bef872f8930c6ee58dce8d13714b672f82c91bdf
Top:    78b780fa0e103667f225f3067ef774228ef0c56a
Author: Ian Kent <ikent@redhat.com>
Date:   2013-04-09 10:23:56 +0800

Refresh of autofs-5.0.7-fix-submount-tree-not-all-expiring.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index f1ec1e5..4106e7f 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -39,6 +39,7 @@
 - document browse option in man page.
 - fix some automount(8) typos.
 - syncronize handle_mounts() shutdown.
+- fix submount tree not all expiring.
 
 25/07/2012 autofs-5.0.7
 =======================
